docs/sources/alerting/configure-notifications/manage-contact-points/integrations/webhook-notifier.md

```diff
@@ -28,13 +28,77 @@ refs:
       destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/template-notifications/
     - pattern: /docs/grafana-cloud/
       destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/template-notifications/
+  configure-contact-points:
+    - pattern: /docs/grafana/
+      destination: /docs/grafana/<GRAFANA_VERSION>/alerting/configure-notifications/manage-contact-points/
+    - pattern: /docs/grafana-cloud/
+      destination: /docs/grafana-cloud/alerting-and-irm/alerting/configure-notifications/manage-contact-points/
 ---
 
-# Configure the webhook notifier for Alerting
+# Configure webhook notifications
+
+Use the webhook integration in contact points to send alert notifications to your webhook.
+
+The webhook integration is a flexible way to integrate alerts into your system. When a notification is triggered, it sends a JSON request with alert details and additional data to the webhook endpoint.
+
+## Configure webhook for a contact point
+
+To create a contact point with webhook integration, complete the following steps.
+
+1. Navigate to **Alerts & IRM** -> **Alerting** -> **Contact points**.
+1. Click **+ Add contact point**.
+1. Enter a name for the contact point.
+1. From the **Integration** list, select **Webhook**.
+1. In the **URL** field, copy in your Webhook URL.
+1. (Optional) Configure [additional settings](#settings).
+1. Click **Save contact point**.
+
+For more details on contact points, including how to test them and enable notifications, refer to [Configure contact points](ref:configure-contact-points).
+
+## Webhook settings
+
+| Option | Description      |
+| ------ | ---------------- |
+| URL    | The Webhook URL. |
+
+#### Optional settings
+
+| Option                            | Description                                                                                                             |
+| --------------------------------- | ----------------------------------------------------------------------------------------------------------------------- |
+| HTTP Method                       | Specifies the HTTP method to use: `POST` or `PUT`.                                                                      |
+| Basic Authentication Username     | Username for HTTP Basic Authentication.                                                                                 |
+| Basic Authentication Password     | Password for HTTP Basic Authentication.                                                                                 |
+| Authentication Header Scheme      | Scheme for the `Authorization` Request Header. Default is `Bearer`.                                                     |
+| Authentication Header Credentials | Credentials for the `Authorization` Request header.                                                                     |
```

Comment:
We can mention here or in a note that the fields for either HTTP Basic Authentication or Authorization Request Header can be set but not both.

Reply:
addressed by https://github.com/grafana/grafana/pull/100650/commits/ca090c5fd1fd25515ae555d309c6eef88b9bbcea